elasticsearch/connection/http_urllib3.py

```diff
@@ -105,7 +114,7 @@ def __init__(self, host='localhost', port=9200, http_auth=None,
                 except AttributeError:
                     ssl_context = None
 
-                if not verify_certs and ssl_context is not None:
+                if not verify_certs:
                     ssl_context.check_hostname = False
```

Comment:
this will fail if `ssl_context` is `None`. Please add tests

Reply:
Ssl_context should never be None, if it is we need to fail loud. 

I’ll raise an exception in that case.